### DISCUSSION TITLE
What happens when an issue disappears from archive.org

### DISCUSSION THREAD

**Discussion by MugUK** [atari-legend/atari-legend#199] [Q&A]
I was reviewing the few ST Action magazines as there was a lot of information missing (no Previews, Budget Reviews or Hints and Tips - all added now). But ST Action 52 no longer exists on Archive.org :(

I downloaded all of the rescans that the guy on AF uploaded to Mega.NZ - are we allowed to upload them to archive.org, if we have his permission. Maybe setup our own Atari Legend section on archive.org to do this?

Or can the AL magazine reader be pointed to his collection on Mega.NZ?

**Comment by photonstorm** [comment]
archive.org is really quite unreliable for long-term linking. I appreciate there are bandwidth costs, but it'd be much better to host these PDFs locally.

**Reply by stgraveyard** [reply]
@Brume-AL @nguillaumin is this feasable? Hosting the mags on AL server? Could we get in trouble for this?

**Reply by nguillaumin** [reply]
I wanted to avoid this as it's large files that may be difficult to handle / backup, and also because archive.org has a nice magazine reader that does not require to load the whole PDF.

It's supposed to be reliable long term as one of the core goal is preservation. Of course there is always the risk of a DMCA takedown, but then if it had been requested to take it down we shouldn't host the content either...

**Reply by nguillaumin** [reply]
I don't think we would be in more trouble than hosting games 😉 But we would need to have some sort of takedown request process.

**Comment by nguillaumin** [comment]
> are we allowed to upload them to archive.org, if we have his permission.

In my view this person has no ownership of the mags themselves, it lies with the original publisher. Presumably they scanned it for preservation/sharing. So I would say ok to upload to archive.org.

> But ST Action 52 no longer exists on Archive.org :(

Do you mean it was there and was removed? Possibly it was never uploaded in the first place?

> Or can the AL magazine reader be pointed to his collection on Mega.NZ?

Technically yes you can put the link to Mega. But you won't be able to point to an individual issue and it's even less long term than archive.org I think. So I would suggest uploading them.
